modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,307 @@
+
+* Support for the `tkn clustertask logs` has been added.
+
+* The YAML merge and variable from `config.yaml` has been removed. In addition, the `release.yaml` file can now be more easily consumed by tools like `kustomize` and `ytt`.
+
+* The support for resource names to contain the dot character (".") has been added.
+
+* The `hostAliases` in the `PodTemplate` specification has been added to the pod-level override of hostname resolution. It ha been achieved by modifying the `/etc/hosts`.
+
+* A variable `$(tasks.status)` has been introduced to access the aggregate execution status of tasks.
+
+* An entry-point binary build for Windows has been added.
+
+
+[id="deprecated-features-1-5_{context}"]
+== Deprecated features
+
+* In the `when` expressions, support for fields written is PascalCase is removed. The `when` expressions only support fields written in lowercase.
++
+[NOTE]
+====
+If you had applied a pipeline with `WhenExpressions` in pipelines `v0.16` (operator `v1.2.x`), you have to reapply it.
+====
+
+* When you upgrade the {pipelines-title} Operator to `v1.5`, the `openshift-client` and the `openshift-client-v-1-5-0` cluster tasks will have the `SCRIPT` parameter. However, the `ARGS` parameter and the `git` resource are removed from the specification of the `openshift-client` cluster task. This is a breaking change, and only those cluster tasks that do not have a specific version in the `name` field of the `ClusterTask` resource will seamlessly upgrade.
++
+To prevent the pipeline runs from breaking, use the `SCRIPT` parameter after the upgrade, as it will move the values previously specified in the `ARGS` parameter into the `SCRIPT` parameter of the cluster task. For example:
++
+[source,yaml,subs="attributes+"]
+----
+...
+- name: deploy
+params:
+  - name: SCRIPT
+    value: oc rollout status <deployment-name>
+  runAfter:
+    - build
+  taskRef:
+    kind: ClusterTask
+    name: openshift-client
+...
+----
++
+
+* When you upgrade from {pipelines-title} Operator `v1.4` to `v1.5`, the profile names in which the `TektonConfig` custom resource is installed will change.
+
++
+.Profiles for `TektonConfig` custom resource
+[cols="1,1,1",options="header"]
+|===
+| Profiles in Pipelines 1.5 | Corresponding profile in Pipelines 1.4 | Installed Tekton components
+| All (_default profile_) | All (_default profile_) | Pipelines, Triggers, Add-ons
+| Basic | Default | Pipelines, Triggers
+| Lite | Basic | Pipelines
+
+|===
+
++
+[NOTE]
+====
+If you used `profile: all` in the `config` instance of the `TektonConfig` custom resource, no change is necessary in the resource specification.
+
+However, if the installed operator is either in the Default or the Basic profile before the upgrade, you must edit the `config` instance of the `TektonConfig` custom resource after the upgrade. For example, if the configuration was `profile: basic` before the upgrade, ensure that it is `profile: lite` after upgrading to Pipelines 1.5.
```

Comment:
operator --> Operator

Reply:
Not applicable in this context.

Reply:
Please see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

Kindly change.